guides/doc-Content_Management_Guide/topics/Importing_Content.adoc

```diff
@@ -102,11 +102,14 @@ endif::[]
 
 .Procedure
 
-. In the {Project} web UI, navigate to *Content* > *Products* and select the product that you want to use, and then click *Create Repository*.
+. In the {Project} web UI, navigate to *Content* > *Products* and select the product that you want to use, and then click *New Repository*.
 . In the *Name* field, enter a name for the repository.
 {ProjectNameX} automatically completes the *Label* field based on what you have entered for *Name*.
 . From the *Type* list, select the type of repository.
 You can select either a repository for RPM files (`yum`), Docker images (`docker`), Files (`file`) or other.
+. If `yum` content type is selected, you can restrict whether the repository is available to a host based on the host's architecture and OS version.
+.. Optional: From the *Restrict to Architecture* list, select the architecture. The repository will be enabled on content hosts with the selected architecture. Select *No restriction*, which is the default value, to make the repository available to all hosts regardless of the architecture.
```

Comment:
```suggestion
. Optional: From the *Restrict to Architecture* list, select the architecture. Select *No restriction*, which is the default value, to make the repository available to all hosts regardless of the architecture.
```
Don't describe behaviour in steps, just the step. You can weave this into the short description if it is necessary.

Reply:
If you do use this text, I'd caution against using the word "enabled." A repo can be restricted to a certain OS but still show up as "enabled" in Repository Sets on the host details screen (even though it won't be available on the host).  We should probably fix this behavior..

Reply:
@melcorr @jeremylenz I've removed the text. Btw, there's also UI text which has the world "enabled".